scripts/Engineering/gui/engineering_diffraction/settings/settings_view.py

```diff
@@ -16,6 +16,7 @@
 class SettingsView(QtWidgets.QDialog, Ui_settings):
```

Comment:
```suggestion
class SettingsView(MantidQt::API::MantidDialog, Ui_settings):
```

Reply:
Might need exposing to python

Reply:
For now, I'm just going to set the flags in the view. The exposure seems unnecessary unless this had to be done in many different places.